recipes/avahi/all/conanfile.py

```diff
@@ -113,7 +113,8 @@ def package_info(self):
             self.cpp_info.components[lib].names["cmake_find_package"] = lib
             self.cpp_info.components[lib].names["cmake_find_package_multi"] = lib
             self.cpp_info.components[lib].libs = [avahi_lib]
-            self.cpp_info.components[lib].includedirs = ["include", os.path.join("include", avahi_lib)]
+            if lib != "common":
```

Comment:
It could be applied to other components too, but this recipe was created 4 years ago and may break users, so let's keep only your change for now.